.github/workflows/dendrite.yml

```diff
@@ -233,7 +233,14 @@ jobs:
       - name: Summarise results.tap
         if: ${{ always() }}
         run: /sytest/scripts/tap_to_gha.pl /logs/results.tap
-
+      - name: Sytest List Maintenance
+        if: ${{ always() }}
```

Comment:
What's the deal with this? I've not seen this before.

Reply:
The list maintenance has already been in Buildkite, but somehow got lost over time.
Not sure if the `if: ${{ always() }}` should be left in or if we want this to fail the build.

Reply:
FWIW I don't think things that are purely informational should fail the pipeline.